_rules/inline-link-in-paragraph-is-distinguishable-be4d0c.md

```diff
@@ -0,0 +1,338 @@
+---
+id: be4d0c
+name: Inline link in paragraph is distinguishable
```

Comment:
Why are you using "paragraph" in these names / files. Why not just say:

```suggestion
name: Inline link is distinguishable
```

I would argue by saying this is an inline link, that's a better qualifier than saying it's a paragraph.